File: src/main/java/com/back/nbe9112team06/domain/adjustresult/repository/AdjustRepository.java
package com.back.nbe9112team06.domain.adjustresult.repository;

import com.back.nbe9112team06.domain.adjustresult.entity.AdjustResult;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AdjustRepository extends JpaRepository<AdjustResult, Integer> {
}

File: src/main/kotlin/com/back/nbe9112team06/domain/adjustresult/repository/AdjustRepository.kt
package com.back.nbe9112team06.domain.adjustresult.repository

import com.back.nbe9112team06.domain.adjustresult.entity.AdjustResult
import org.springframework.data.jpa.repository.JpaRepository

interface AdjustRepository : JpaRepository<AdjustResult, Int>
